frb_codegen/src/lib.rs

```diff
@@ -4,14 +4,16 @@ use std::path::Path;
 use log::info;
 use pathdiff::diff_paths;
 
-use crate::commands::ensure_tools_available;
-pub use crate::config::RawOpts as Opts;
 use crate::ir::*;
 use crate::others::*;
 use crate::utils::*;
 
-mod commands;
 mod config;
+pub use crate::commands::ensure_tools_available;
+pub use crate::config::parse as config_parse;
+pub use crate::config::Opts;
```

Comment:
Do we really need to *pub* use these, i.e. expose it at this location to all external users?

Reply:
Without these pubs, I didn't find way to let the method in `main.rs` to get access under `main+lib` framework. Or is there a better way?

Reply:
You are right. Agree for opts and config_parse